test/kubemark/start-kubemark.sh

```diff
@@ -214,6 +214,25 @@ function create-and-upload-hollow-node-image {
   echo "Created and uploaded the kubemark hollow-node image to docker registry."
 }
 
+# Use bazel rule to create a docker image for hollow-node and upload
+# it to the appropriate docker container registry for the cloud provider.
+function create-and-upload-hollow-node-image-bazel {
```

Comment:
Most of this function's logic is similar to `create-and-upload-hollow-node-image`. The main difference is we're calling bazel here and make there. Could you change it to switch b/w both modes (based on a flag or sth)>

Reply:
it looks for `KUBEMARK_BAZEL_BUILD`, if that's what you meant?

Reply:
Yup, saw that.. But you still don't seem to need 2 functions pretty much replicating the same logic.

Reply:
My intend is to swap out the entire function later on, seems this is clearer than embed more if/else into the existing function?

Reply:
Sgtm